app/javascript/modCenter/singleArticle/singleArticle.test.jsx

```diff
@@ -29,15 +47,32 @@ describe('<SingleArticle />', () => {
       />,
     );
 
-  it('should have no a11y  violations', async () => {
+  const renderSingleArticle2 = (article = testArticle2) =>
```

Comment:
Instead of making the `testArticle2` a default parameter, consider just doing all this within the test. Also, `testArticle2` can be mutated as well as `testArticle1`. Consider creating a `getArticle1()` (needs a better name) etc. so that each test gets it's own test article.

Reply:
👍